components/ciphertool/src/main/java/org/wso2/ciphertool/CipherTool.java

```diff
@@ -147,11 +147,7 @@ private static void loadXpathValuesAndPasswordDetails() {
                 Utils.loadProperties(System.getProperty(Constants.CIPHER_TEXT_PROPERTY_FILE_PROPERTY));
         for (Object key : cipherTextProperties.keySet()) {
             String passwordAlias = (String) key;
-            if (configFileXpathMap.containsKey(passwordAlias)) {

```

Comment:
how could this resolve the non-xml case?

Reply:
The issue was that it was searching the key in cipher-tool.properties for a entry where we don't define for non-xml fields. (for manual encryption). So this fix allows such these cases where we don't define the entry in cipher-tool.properties

Reply:
:+1: